bioptim/dynamics/ode_solver.py

```diff
@@ -342,7 +342,7 @@ def integrator(self, ocp, nlp) -> list:
                 "p": nlp.controls.cx,
                 "ode": nlp.dynamics_func(nlp.states.cx, nlp.controls.cx, nlp.parameters.cx),
             }
-            ode_opt = {"t0": 0, "tf": nlp.dt}
+            ode_opt = {"t0": 0, "tf": nlp.dt, "abstol": 1e-10, "reltol": 1e-10}
```

Comment:
I think, this makes the tests to take ages?

Reply:
The resolution with CVODES is not included in the tests. but yes i've commited a local trial.